config/smart_answers/check_benefits_financial_support_data.yml

```diff
@@ -636,10 +636,22 @@
     - name: help_to_save
       title: Help to Save
       description: <p>You can get a bonus of 50p for every £1 you save over 4 years through Help to Save if you get certain benefits.</p>
+      condition: eligible_for_help_to_save?
       url_text: Find out more about Help to Save
       url: https://www.gov.uk/get-help-savings-low-income
       countries:
         - england
         - wales
         - scotland
         - northern-ireland
+    - name: disabled_facilities_grant
+      title: Disabled Facilities Grant
+      description: <p>You could get a grant from your local council to make changes to your home if you’re disabled or you live with someone who is.</p>
+      condition: eligible_disabled_facilities_grant?
+      url_text: Check if you’re eligible for a Disabled Facilities Grant
+      url: https://www.gov.uk/disabled-facilities-grants
```

Comment:
ah so that is in the following chunk of YAML, they use the same method but different content from the YAML file. I updated the name of the following bit of YAML to be called `- name: disabled_facilities_grant_ni` to make it a bit clearer and avoid a name clash!

Reply:
Gotcha!